src/Translate.HOC.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default component => {
```

Comment:
Could you add a displayname to the component for easier debugging?

See https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging

Reply:
Ah, that's a good-read, thanks for that, will add the display name.